Examples/test-suite/go/Makefile.in

```diff
@@ -182,12 +182,3 @@ clean:
 	rm -f import_stl_a.go import_stl_a.gox
 	rm -f import_stl_b.go import_stl_b.gox
 	rm -rf gopath
-
-cvsignore:
-	@echo '*_gc.c *_wrap.* *.so *.dll *.exp *.lib'
-	@echo Makefile
-	@echo mod_a.go mod_b.go imports_a.go imports_b.go
-	@echo clientdata_prop_a.go clientdata_prop_b.go
-	@echo multi_import_a.go multi_import_b.go
-	@echo packageoption_a.go packageoption_b.go packageoption_c.go
-	@for i in ${CPP_TEST_CASES} ${C_TEST_CASES}; do echo $$i.go; done
```

Comment:
Hmm, you seem to have somehow managed to include a cleanup commit from William from somewhere into this PR (I don't see that commit in master though).